catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/miro/MiroImageApiURL.scala

```diff
@@ -0,0 +1,19 @@
+package uk.ac.wellcome.platform.transformer.transformers.miro
+
+trait MiroImageApiURL {
+  def buildImageApiURL(miroID: String, templateName: String): String = {
+    val iiifImageApiBaseUri = "https://iiif.wellcomecollection.org"
+
+    val imageUriTemplates = Map(
+      "thumbnail" -> "%s/image/%s.jpg/full/300,/0/default.jpg",
+      "info" -> "%s/image/%s.jpg/info.json"
+    )
+
+    val imageUriTemplate = imageUriTemplates.getOrElse(
+      templateName,
+      throw new Exception(
```

Comment:
NoSuchElementException would be more specific, and a bit nicer.

Reply:
Agree, but that's just a copy paste of the current logic somewhere else. I actually wonder what happens in that case...does it go in the DLQ? it should have a test for it probably..